lib/ansible/modules/windows/win_share.ps1

```diff
@@ -111,26 +111,34 @@ Function NormalizeAccounts
 
 $result = @{
     changed = $false
+    actions = @() # More for debug purposes
```

Comment:
Well, I try to avoid this personally. But you are right, you'd need a special function if you want to make this more efficient. (Which is what we did for e.g. Add-Warning)